manifests/public.yaml

```diff
@@ -326,7 +326,7 @@ spec:
     - name: VELUM_INTERNAL_API_PASSWORD_FILE
       value: /var/lib/misc/infra-secrets/velum-internal-api-password
     - name: LDAP_HOST
-      value: "127.0.0.1"
+      value: "ldap.infra.caasp.local"
```

Comment:
So, during the highstates the ldap certificate gets generated by salt:

https://github.com/kubic-project/salt/blob/f29c60a90ecccf49089255267a119ede4c1f6032/salt/ldap/init.sls#L7-L12

I see that after this certificate gets recreated, from a running cluster:

```
            X509v3 Subject Alternative Name: 
                DNS:caasp-admin, DNS:caasp-admin.infra.caasp.local, DNS:be19dde4e72240bdbf9939d4720edec5, DNS:be19dde4e72240bdbf9939d4720edec5.infra.caasp.local, IP Address:10.17.1.0
```

I think we should also add `ldap.infra.caasp.local` to `ldap/init.sls` so Velum will correctly validate the certificate after the bootstrap has finished, since with the current situation I think Velum should fail to connect to LDAP after the bootstrap has finished and after we have regenerated the ldap certificate.

Reply:
https://github.com/kubic-project/salt/pull/396